### DISCUSSION TITLE
Rounds modded on linux steam deck problem

### DISCUSSION THREAD

**Discussion by ModLover123** [ebkr/r2modmanPlus#988] [Q&A]
when i use r2modman i get this text:

Error
ROUNDS compatibility data does not exist in Steam's specified location
Failed to find directory: /run/media/mmcblk0p1/steamapps/compatdata/1557740

Suggestion
If this happened, it is very likely that you did not start the game at least once. Please do it.

I did start the game and played for more than an hour.
CAn anyone please tell me how I could get this to work.

**Comment by ebkr** [comment]
The game needs to be installed on your main drive
